sbt-scalaxb/src/sbt-test/sbt-scalaxb/address/build.sbt

```diff
@@ -1,13 +1,38 @@
-val scalaXml = "org.scala-lang.modules" %% "scala-xml" % "1.1.1"
-val scalaParser = "org.scala-lang.modules" %% "scala-parser-combinators" % "1.1.1"
+val scalaXml = Def.setting(
+  scalaBinaryVersion.value match {
+    case "2.10" =>
+      Nil
+    case "2.11" | "2.12" =>
+      Seq("org.scala-lang.modules" %% "scala-xml" % "1.1.1")
+    case _ =>
+      Seq("org.scala-lang.modules" %% "scala-xml" % "2.2.0")
+  }
+)
+val scalaParser = Def.setting(
+  scalaBinaryVersion.value match {
+    case "2.10" =>
+      Nil
+    case "2.11" | "2.12" =>
+      Seq("org.scala-lang.modules" %% "scala-parser-combinators" % "1.1.1")
+    case _ =>
+      Seq("org.scala-lang.modules" %% "scala-parser-combinators" % "2.3.0")
+  }
+)
 val jaxbApi = "javax.xml.bind" % "jaxb-api" % "2.3.1"
 
 lazy val root = (project in file(".")).
   enablePlugins(ScalaxbPlugin).
   settings(
-    scalaVersion := "2.12.18",
+    crossScalaVersions := Seq(
+      "2.10.7",
+      "2.11.12",
+      "2.12.18",
+      "2.13.12",
+    ),
```

Comment:
could not add Scala 3 due to https://github.com/eed3si9n/scalaxb/blob/64033cfa3ce33591815eb9907f84b648b95301bd/build.sbt#L98

(sbt 1.2 does not support Scala 3)

Reply:
Let's use sbt 1.5.8 then

Reply:
done